clap-v3-utils/src/keypair.rs

```diff
@@ -1175,7 +1163,7 @@ pub fn keypair_from_seed_phrase(
     let keypair: Keypair =
         encodable_key_from_seed_phrase(keypair_name, skip_validation, derivation_path, legacy)?;
     if confirm_pubkey {
-        confirm_keypair_pubkey(&keypair);
+        confirm_encodable_keypair_pubkey(&keypair, "keypair");
```

Comment:
```suggestion
        confirm_encodable_keypair_pubkey(&keypair, "pubkey");
```
Otherwise this is a slight behavior change, right?
I don't feel strongly about preserving "pubkey", but it seems like `keypair_from_path()` and `keypair_from_seed_phrase()` should remain consistent.

Reply:
Oops, this was a mistake. Thanks for the catch!